client/src/main/java/io/pravega/client/netty/impl/RawClient.java

```diff
@@ -9,10 +9,10 @@
  */
 package io.pravega.client.netty.impl;
 
+import io.pravega.auth.AuthenticationException;
```

Comment:
There are a bunch of gratuitous changes to the order of imports in this PR. Please avoid such unnecessary formatting changes, which are unrelated to the issue being resolved here.

Reply:
This exception is moved to a different package because of change in the `AuthHandler` interface. These were done as part of a review comment. With Pravega settings the IDE sorts the imports alphabetically. Because of this the location of this import changed. Do let me know if you think this change should still be inline.